sqla_yaml_fixtures.py

```diff
@@ -1,13 +1,19 @@
 __version__ = (0, 1, 2)
 
-import functools
+try:
+    # Python 3
+    from functools import lru_cache
+except ImportError:
+    # For Python 2: 
+    # use https://pypi.python.org/pypi/backports.functools_lru_cache
```

Comment:
the package information is already `setup.py` so i dont see why it should be mentioned here.